tinygrad/tensor.py

```diff
@@ -246,6 +246,7 @@ def flip(self, axis, *args) -> Tensor: return mlops.Flip.apply(self, axis=[x if
   def shrink(self, arg:Tuple[Tuple[int, int], ...]) -> Tensor: return mlops.Shrink.apply(self, arg=arg) if any(x != (0,s) for x,s in zip(arg, self.shape)) else self
   def pad(self, arg: Tuple[Tuple[int, int], ...], value:float=0) -> Tensor:
     ret = mlops.Pad.apply(self, arg=arg) if any(x != (0, 0) for x in arg) else self
+    if isinf(value): return ret + copysign(1,value)*(1/mlops.Pad.apply(Tensor.full(self.shape, value), arg=arg))
```

Comment:
can't you just do `copysign(1, value)/mlops.Pad...` here?

Reply:
Assumed it will be more readable as I was dealing with 1/inf = 0, formatted it now.